### ISSUE TITLE
failed for transferring from gromacs XTC trajectory to yasp TRZ trajectory

### ISSUE THREAD

**Issue by jluchemlib** [MDAnalysis/mdanalysis#302]
Dear all,

I am a new user of MDAnalysis.
I am trying to convert gromacs XTC trajectory to yasp TRZ trajectory. My input python script is like this:

```python
#!/usr/bin/env python

from MDAnalysis.tests.datafiles import *
from MDAnalysis import Universe, Writer

import os.path

XTC = "traj.xtc"
GRO = "ions.gro"
root, ext = os.path.splitext(os.path.basename(XTC))
trzname = root + '.trz' 

u = Universe(GRO, XTC)
w = Writer(trzname, u.trajectory.numatoms)

for ts in u.trajectory:
    w.write(ts)
    print "Converted frame %d" % ts.frame
w.close()
print "Converted %r --> %r" % (XTC, trzname)
```

I just modified the example of `dcd2xtc`.
But I got an error message that:

```
  File "mdanalysis.py", line 30, in <module>
    w.write(ts)
  File "/usr/local/lib/python2.7/dist-packages/MDAnalysis/coordinates/base.py", line 963, in write
    return self.write_next_timestep(ts)
  File "/usr/local/lib/python2.7/dist-packages/MDAnalysis/coordinates/TRZ.py", line 594, in write_next_timestep
    out['box'] = self.convert_pos_to_native(ts._unitcell, inplace=False)
ValueError: could not broadcast input array from shape (3,3) into shape (9)
```

It seems that because the box dimensions in XTC is 3*3 but in TRZ is 9.

How could I fix the problem?

Thank you.

Best regards,
Bin Li

**Comment by hainm** [comment]
@jluchemlib you can use markdown syntax to have nice python code 

https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#code
```python
#!/usr/bin/env python

from MDAnalysis.tests.datafiles import *
from MDAnalysis import Universe, Writer

import os.path

XTC = "traj.xtc"
GRO = "ions.gro"
root, ext = os.path.splitext(os.path.basename(XTC))
trzname = root + '.trz'

u = Universe(GRO, XTC)
w = Writer(trzname, u.trajectory.numatoms)

for ts in u.trajectory:
w.write(ts)
print "Converted frame %d" % ts.frame
w.close()
print "Converted %r --> %r" % (XTC, trzname)
```

**Comment by orbeckst** [comment]
@hainm – I added the formatting to @jluchemlib 's report. I also think that this is one of the very pretty features on github and encourage it :-).

**Comment by jluchemlib** [comment]
@orbeckst @hainm thank you all. This is my first post on github and I don't really know that the syntax format before.

**Comment by orbeckst** [comment]
Don't worry, thanks for contributing a bug report. That is very valuable. The way that it is formatted is really not that important, it is the content that counts. And now in the future you know that you can also make it look

Am Jun 12, 2015 um 2:01 schrieb Bin Li <notifications@github.com>:

**Comment by richardjgowers** [comment]
Should be fixed through PR #311
